Description: Prevent an error of fontset-pixel-size for Emacs 23.
Bug-Debian: http://bugs.debian.org/477177
Author: Tatsuya Kinoshita <tats@debian.org>

	* poem-e20.el (fontset-pixel-size): Don't cause an error. Workaround
	  for Emacs 23.

--- apel-10.7+0.20080906.orig/poem-e20.el
+++ apel-10.7+0.20080906/poem-e20.el
@@ -25,6 +25,7 @@
 ;;; Code:
 
 (defun fontset-pixel-size (fontset)
+ (condition-case nil
   (let* ((info (fontset-info fontset))
 	 (height (aref info 1))
 	 )
@@ -32,7 +33,8 @@
 	  ((string-match "-\\([0-9]+\\)-" fontset)
 	   (string-to-number
 	    (substring fontset (match-beginning 1)(match-end 1))))
-	  (t 0))))
+	  (t 0)))
+  (error 0)))
 
 
 ;;; @ character set
